### PR TITLE
#30172 Add the missing validation translations for the Luxembourgish …

### PR DESCRIPTION
**Pull request by evxzibit**
…locale

| Q             | A
| ------------- | ---
| Branch?       | master for features / 3.4 up to 4.2 for bug fixes 
| Bug fix?      | yes
| New feature?  | no 
| BC breaks?    | no     
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets | #30172  
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Add the missing validation translations for the Luxembourgish locale

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf [hunk 1]
```diff
@@ -322,6 +322,18 @@
                 <source>This is not a valid UUID.</source>
                 <target>Dëst ass keng gëlteg UUID.</target>
             </trans-unit>
+            <trans-unit id="84">
+                <source>This value should be a multiple of {{ compared_value }}.</source>
+                <target>Dëse Wäert sollt e puer vun  {{ compared_value }} sinn.</target>
```
Comment (javiereguiluz) [inline src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf]: There are two white spaces after `puer vun`. Let's remove one of them.

**Review by javiereguiluz — APPROVED** [review]
@evxzibit thanks a lot and congrats on your first Symfony contribution!

I left a very minor comment about a minor issue.

File: src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf (hunk 1 above)
Comment (evxzibit) [reply src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf]: fixed it thank you!

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @evxzibit.

**Pull request merged**